SpeechResponder/eddi.es.json

```diff
@@ -1559,6 +1559,15 @@
       "name": "Ship swapped",
       "description": "Triggered when you swap a ship"
     },
+    "Ship targeted": {
+      "enabled": true,
+      "priority": 3,
+      "responder": true,
+      "script": "{_ Ship targeted }\r\n{_ Triggered when the player selects a target }\r\n\r\n\r\n{_ Context }\r\n{SetState('eddi_context_last_subject', 'ship')}\r\n{SetState('eddi_context_last_action', 'targeted')}\r\n\r\n",
+      "default": true,
+      "name": "Ship targeted",
+      "description": "Triggered when the player selects a target"
+    },
```

Comment:
It's possible to create a more involved script for this, but there are some issues with such a script... it's quite easy to switch targets, switch back, and hear the same info that you heard 15 seconds earlier. It's obviously not a problem if you just use this event for module targeting, but it could be a significant annoyance for someone trying to use this script for more than module targeting. Thoughts?

Reply:
Nuisance is in the eye of the beholder. I tend to go minimal and let the community drive that bus.

Reply:
Understandable. All I'm asking is if you've given some thought to minimizing the nuisance for someone who wants to go a different direction with this event.